common/app/views/fragments/amp/stylesheets/main.scala.html

```diff
@@ -246,6 +246,12 @@
         list-style: none;
     }
 
+    @* temporarily hiding these for after launch *@
+    .inline-expand-image, .content__dateline-lm , .block-share, .witness-cta-wrapper, .block-share {
+        display: none;
+        visibility: hidden;
```

Comment:
You shouldn't need both? `display: none` removes the node from the render tree entirely.